docs/_site/index.html

```diff
@@ -0,0 +1,231 @@
+
+<hr />
+
+<p>The <strong>DDK Core</strong> is a library of CDK constructs that you can use to build data workflows and modern data architecture on AWS, following our best practice. The DDK Core is modular and extensible, if our best practice doesn’t work for you, then you can update and share your own version with the rest of your organisation by leveraging a private <strong>AWS Code Artifact</strong> repository.</p>
+
+<p>You can compose constructs from the DDK Core into a <strong>DDK App</strong>.  Your DDK App can also add contain constructs from the CDK Framework or the <a href="https://docs.aws.amazon.com/cdk/api/latest/docs/aws-construct-library.html">AWS Construct Library</a>.</p>
+
+<p>For a detailed walk-through, check out our <a href="https://catalog.us-east-1.prod.workshops.aws/workshops/3644b48b-1d7c-43ef-a353-6edcd96385af/en-US">Workshop</a> or
+take a look at <a href="https://github.com/aws-samples/aws-ddk-examples">examples</a>.</p>
+
+<h2 id="install">Install</h2>
+
+<p>You can install or update the AWS DDK library from the following:</p>
+<ul>
+  <li><a href="https://pypi.org/project/aws-ddk-core/">PyPi</a>
+    <div class="language-shell highlighter-rouge"><div class="highlight"><pre class="highlight"><code>pip <span class="nb">install </span>aws-ddk-core
+</code></pre></div>    </div>
+  </li>
+  <li><a href="https://www.npmjs.com/package/aws-ddk-core">NPM</a>
+    <div class="language-shell highlighter-rouge"><div class="highlight"><pre class="highlight"><code>npm <span class="nb">install </span>aws-ddk-core
+</code></pre></div>    </div>
+  </li>
+</ul>
+
+<h3 id="build-data-pipelines">Build Data Pipelines</h3>
+
+<p>One of the core features of DDK is ability to create Data Pipelines. A DDK <a href="https://awslabs.github.io/aws-ddk/release/stable/api/core/stubs/aws_ddk_core.pipelines.DataPipeline.html">DataPipeline</a> 
+is a chained series of stages. It automatically “wires” the stages together using 
+<a href="https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-rules.html">AWS EventBridge Rules</a> .</p>
+
+<p>DDK comes with a library of stages, however users can also create their own based on their use cases, 
+and are encouraged to share them with the community.</p>
+
+<p>Let’s take a look at an example below:</p>
+
+<div class="language-python highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="p">...</span>
+
+<span class="n">firehose_s3_stage</span> <span class="o">=</span> <span class="n">KinesisToS3Stage</span><span class="p">(</span>
```

Comment:
This part is autogenerated from `index.markdown` I assume? Do we need to gave these autogenerated HTML files in the repo? Can we at least use `.gitattributes` to [mark them as such](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github)?